packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.0.4] - June 10, 2021 //TODO Edit data
```

Comment:
Make sure we have a proper date here. If needed we can change it later.

Reply:
I have changed it to the current date.